content/300-guides/200-deployment/201-serverless/500-deploy-to-netlify.mdx

```diff
@@ -237,44 +67,45 @@ You can now create your Netlify site and configure continuous deployments for it
 
    1. Select `Create & configure a new site`.
    2. If prompted to select your team, pick the one in which Netlify will create the site.
-   3. Give the site a unique name.
-   4. Authorize Netlify with your GitHub account.<br />
-      Netlify uses your GitHub account to configure notification webhooks and deploy keys. It uses these to configure the continuous integration for the app.
-   5. _Your build command_: `npm run build`
-   6. _Directory to deploy_: `public`
+   3. Give the site a unique name, or use the default. It can be updated in the future.
+   4. Authorize Netlify with your GitHub account. Netlify uses your GitHub account to configure notification webhooks and deploy keys. It uses these to configure the continuous integration for the app.
+   5. _Your build command_: The command you use to build your project. Common examples include `npm run build`, `yarn run build`, `next build`, etc.
+   6. _Directory to deploy_: Generally, the directory that your `build` command outputs to. Common examples include `dist/`, `build/`, `.next`, etc.
+   7. Allow Netlify to create a `netlify.toml` file.
 
-### Result
+Netlify has now setup your site and connected your github repo to it! You should now be able to use `netlify open` to view the Admin page of your newly created Site.
 
-Check that the Netlify CLI shows the following:
+You can open your app now, but you may see errors when you use options that connect to the database. This is because Prisma will look to connect to your database, but if your connection string is setup as an environment variable [Netlify will not recognize it by default](https://docs.netlify.com/environment-variables/get-started/#work-with-env-files).
```

Comment:
What does "not recognize be default" mean? Isn't the problem that the env var is not stored on Netlify, and the `.env` file is not in the repo and also not read on Netlify?

Reply:
As far I as understand, the build step that Netlify uses ignores/excludes `.env` files so when you go to deploy your app and expect Prisma to "just work", it won't. 💀 

So specific steps need to be taken to upload env vars. It's not a big lift.

Reply:
`.env` files hopefully are not even in your GitHub repository.

Yeah, then we should say "If you Prisma schema tries to load the connection string from an environment variable, you have to create it first in Netlify" or something like that.